articles/getting-started-with-quasar-framework/index.md

```diff
@@ -0,0 +1,295 @@
+```bash
+$ npm uninstall -g vue-cli 
+```
+if you're using yarn, run
+```bash
+$ yarn global remove vue-cli
+```
+Now install Vue CLI (4+) with the command below.
+```bash
+$ npm install -g @vue/cli
+```
+if you're using yarn, run
+```bash
+$ yarn global add @vue/cli
+```
+To create a project run the command below
+```bash
+$ vue create my-app
+```
+Make sure to select babel from the vue CLI features list prompt while installing.
+```bash
+❯ Default ([Vue 2] babel, eslint) 
+```
+To add Vue CLI Quasar Plugin, navigate to the created project and run `vue add quasar` to add the quasar plugin
+```bash
+$ cd my-app
+$ vue add quasar
+```
+The CLI will prompt you to replace some existing files, you should accept this and move to pick features you'll need for the quasar plugin.
+
+This may not be the best approach to install quasar, because you'll not have access to some of the features provided by quasar CLI. Besides, quasar was not tested with these plugins.
+
+3. Installing using Quasar CLI
+
+Before we install quasar, first check if Quasar CLI is globally installed on your computer. Using the terminal run:
+```bash
+$ quasar -v
+```
+If you get a "command not found", run the following command to install it.
+```bash
+$ npm install -g @quasar/cli
+```
+if you're using yarn, run
+```bash
+$ yarn global add @quasar/cli
+```
+
+### Create a hello-world-app with quasar
+
+To create a new quasar app run the command below. `hello-world-app` is the name of your project.
+```bash
+$ quasar create hello-world-app
+```
+Hit Enter to accept the default project details or you can edit to your liking.
+
+Using the arrow keys select the first CSS preprocessor:
+```bash
+❯ Sass with SCSS syntax (recommended) 
+  Sass with SCSS syntax (recommended)
+  Sass with indented syntax (recommended)
+  Stylus (deprecated)
+  None (the others will still be available)
+```
+Select Auto-import-in-use Quasar Components and hit enter
+```bash
+? Pick a Quasar components & directives import strategy
+❯ * Auto-import in-use Quasar components & directives
+    - also treeshakes Quasar; minimum bundle size
+  * Import everything from Quasar
+    - not treeshaking Quasar; biggest bundle size    
+```
+Then press the spacebar to unselect the ESLint feature, we won't be needing it here. Hit enter.
+```bash
+❯ (*) ESLint (recommended)
+```
+Finally using arrow keys select yarn, this is what we will use during development, you can use npm if you like.
+```bash
+❯ Yes, use Yarn (recommended)
+```
+
+Navigate to the created `hello-world-app` and serve the app by running the commands:
+```bash
+$ cd hello-world-app
+$ quasar dev
+```
+After serving the app, it  opens the URL http://localhost:8080/#/ in your browser to view the app.
+You'll see quasar logo and the name quasar in the center of the page.
+
+Open the created `hello-world-app` folder in your editor of choice.
+
+Navigate to the `src` folder, open App.vue you'll see a div with and id of q-app
```

Comment:
Write inline file and folder names and code excerpts as inline code blocks. eg . `App.vue`. Revise your document fixing similar issues.